livekit-plugins/livekit-plugins-openai/livekit/plugins/openai/realtime/realtime_model.py

```diff
@@ -38,6 +38,11 @@
     "metrics_collected",
 ]
 
+OpenAIModel = Literal[
+    "gpt-4o-realtime-preview-2024-12-17", "gpt-4o-mini-realtime-preview-2024-12-17"
```

Comment:
this should also allow the unversioned ones. there's actually six valid values

<img width="345" alt="image" src="https://github.com/user-attachments/assets/9f50b571-4fdb-408e-b955-27973be71ac0" />

Reply:
good point. the default should point to unversioned